zproject/prod_settings_template.py

```diff
@@ -588,6 +588,9 @@
 ################
 ## Miscellaneous settings.
 
+## How long outgoing webhook requests time out after
```

Comment:
"How long servers have to respond to outgoing webhooks"?

Reply:
Sure!